runtime/extensions/CMakeLists.txt

```diff
@@ -3,49 +3,30 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 include(FetchContent)
 
-if(ENABLE_LIGHTNING_KOKKOS)
-    set(CMAKE_CXX_EXTENSIONS OFF)
-
-    set(LIGHTNING_GIT_TAG "main" CACHE STRING "GIT_TAG value to build Lightning-Kokkos")
+set(LIGHTNING_GIT_TAG "main" CACHE STRING "GIT_TAG value to build Lightning-Kokkos")
+set(source_files "StateVectorDynamicCPU.cpp")
+FetchContent_Declare(
+    pennylane_lightning
+    GIT_REPOSITORY https://github.com/PennyLaneAI/pennylane-lightning.git
+    GIT_TAG      ${LIGHTNING_GIT_TAG}
+)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON) # build with -fPIC
+set(ENABLE_KOKKOS OFF CACHE BOOL "Enable Kokkos")
+FetchContent_MakeAvailable(pennylane_lightning)
+
+add_library(lightnings_interface INTERFACE)
```

Comment:
I did use `INTERFACE` to avoid building any artifacts by just defining the properties required for building the `rt_backend` shared lib that includes all requested devices. We can change this to add the feature of loading/unloading backend simulators to `rt_capi` at runtime in future; within this feature, we'd end-up building `liblightning` `liblightning_kokkos` and then they can be loaded to `rt_capi` execution context manager at runtime...

Reply:
Thanks, was just curious :)